cider-dap/src/main.rs

```diff
@@ -4,18 +4,18 @@ mod error;
 use adapter::MyAdapter;
 use dap::events::{ExitedEventBody, StoppedEventBody, ThreadEventBody};
 use dap::responses::{
-    ContinueResponse, SetBreakpointsResponse, SetExceptionBreakpointsResponse,
-    StackTraceResponse, ThreadsResponse,
+    ContinueResponse, ScopesResponse, SetBreakpointsResponse,
+    SetExceptionBreakpointsResponse, StackTraceResponse, ThreadsResponse,
 };
 use error::MyAdapterError;
 
 use dap::prelude::*;
 use error::AdapterResult;
 use slog::{info, Drain};
-use std::fs::File;
 use std::fs::OpenOptions;
 use std::io::{stdin, stdout, BufReader, BufWriter, Read, Write};
 use std::net::TcpListener;
+use std::vec;
```

Comment:
This import isn't necessary since `Vec` and `vec!` are included in the rust prelude